index.html

```diff
@@ -57,6 +57,172 @@ <h1 id="head-1">
           </li>
         </ul>
       </section>
+
+      <!-- --------------------------- Project-Section--------------- -->
+
+      <div class="Proj-container">
+        <h1>
+          My Recent<br />
+          Works
+        </h1>
+        <div class="proj-card">
+          <img src="imege/web-imege01.webp" />
+
+          <div class="card-content">
+            <h2>
+              Multi-Post Stories<br />
+              Gain+Glory
+            </h2>
+            <ul class="tools">
+              <li>Ruby on rails</li>
+              <li>Css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="See-Project">
+              <a href="#"></a>See Project
+            </button>
+          </div>
+        </div>
```

Comment:
- Good job designing your project cards but it still has a few issues that need to be taken care of
- [ ] Kindly use the same images as in the template so as to follow the template design.
- [ ] It would be great to apply the correct font family and font size of the project heading 👍 
- [ ] Kindly style the button as shown in the template and apply the different button interactions so as to improve the user interface 👍 
- Your design:
![Annotation 2022-03-18 202551](https://user-images.githubusercontent.com/85113567/159075646-9b61b729-60cc-4259-b63a-0318450f010c.jpg)
- Figma design:
![Annotation 2022-03-18 202536](https://user-images.githubusercontent.com/85113567/159075755-a0a46455-10ba-406e-a253-81acd1a3f04b.jpg)

- Button interactions:
![Annotation 2022-03-18 210529](https://user-images.githubusercontent.com/85113567/159076029-63198f08-eff3-4fb1-8ee8-af72d6ad108a.jpg)